Formula/root.rb

```diff
@@ -118,5 +125,10 @@ def caveats; <<~EOS
     EOS
     assert_equal "\nProcessing test.C...\nHello, world!\n",
                  shell_output("/bin/bash test.bash")
+
+    if build.with? "python"
+      ENV["PYTHONPATH"] = lib/"root"
```

Comment:
hmm seems like this shouldn't be needed

Reply:
Check the directory layout. It’s... unique. Unless I’m missing a deeper meaning behind your comment?

Reply:
Right. I mean why aren't they getting installed into a location that our pythons can find them without monkeying around? Can we pass our site directory to cmake or something?

Reply:
I presumed from the formula's complexity & build time it came from science and there was some solid reason for it that I'd cry if I went looking for 🙈. It seems to be expected on upstream's part; the script in `bin` literally seems to purely exist to compensate for their weird directory structure.

Reply:
probably some way to tweak it but me is happy

Reply:
It is confusing though, especially if you miss the caveats. This PR was prompted in part by having to build it locally & do some testing this morning after an IRL friend text me like hey you were involved with Homebrew why isn't this working 🙃.

Reply:
The other thing here is that now we're enforcing the use of Homebrew's `python` and `python3` instead of using requirements things like `which("python")` could simply be replaced with `Formula["python"].opt_libexec/"bin/python"` as desired, etc. But I left the style as similar as possible here, whilst also trying to compact it & make it a little less grim to look at 😓.